sound/soc/sof/core.c

```diff
@@ -367,9 +367,17 @@ void snd_sof_shutdown(struct device *dev)
 }
 EXPORT_SYMBOL(snd_sof_shutdown);
 
+static const struct dev_pm_ops sof_platform_pm = {
```

Comment:
I don't know if this is correct. I would tend to think that the ACPI/PCI device are at the top in the device hierarchy, so wondering if the pm operations should be done at the highest level?

Reply:
@plbossart OK, let me change it back in that case.

Reply:
@lgirdwood runtime_enable for the pci device shows as forbidden.

I could use some guidance here. Is it OK to set the PM callbacks for the sof-audio device?

Reply:
in the skylake driver the pm_ops are set in the pci driver, see sound/soc/intel/skylake/skl.c